script.js

```diff
@@ -230,16 +227,13 @@ async function justBack() {
 }
 
 async function goStore() {
-    await delayUpdate(elements.text, "Going To Store", 100);
-    await delayUpdate(elements.text, "Going To Store.", 100);
-    await delayUpdate(elements.text, "Going To Store..", 100);
-    await delayUpdate(elements.text, "Going To Store...", 100);
-    await delayUpdate(elements.text, "", 100);
-    elements.shopUI.style.visibility = "visible";
-    elements.controlsForMonsters.style.visibility = "hidden";
-    elements.text.innerText = "";
-    buttons.navigation[1].disabled = true;
+    if (buttons.navigation[1].classList.contains("shopUI")) {
+        elements.shopUI.classList.remove("shopUI")
+    } else {
+        elements.shopUI.classList.add("shopUI")
```

Comment:
having a problem here because i want to check if the goStore button contains the shopUI but i think im doing wrong